GVRf/Framework/framework/src/main/java/org/gearvrf/script/GVRScriptManager.java

```diff
@@ -218,7 +219,8 @@ public GVRScriptFile loadScript(GVRAndroidResource resource, String language) th
         if (language.equals(LANG_LUA)) {
             script = new GVRLuaScriptFile(mGvrContext, resource.getStream());
         } else if (language.equals(LANG_JAVASCRIPT)) {
-            script = new GVRJavascriptScriptFile(mGvrContext, resource.getStream());
+            // Commented out for v8ScriptingEngine
+            //script = new GVRJavascriptScriptFile(mGvrContext, resource.getStream());
```

Comment:
Initial development, I had to comment out references to the Mozilla Rhino engine to build V8 but still tried to enable Rhino or V8 with a Boolean.  This became impractical and thus created GVRJavascriptV8File.java. to replace GVRJavascriptScriptFile and no longer needed GVRScriptManager.  We could try to run both JavaScript engines  - X3D uses V8, others use Mozilla Rhino.  This might be the fastest way to get this back to working and revisit this issue later.  Otherwise, this seems like a bit of a re-coding effort.